rdtools/__init__.py

```diff
@@ -1,4 +1,4 @@
-from energy_normalization import normalize_with_sapm
```

Comment:
Yep, this is for all functions to be in the main rdtools namespace, i.e., `rdtools.function`. I can add the latest degradation functions to __init__ before the pull-request merge. I just need to merge the latest master into this branch.